lib/validateWebhook.js

```diff
@@ -0,0 +1,15 @@
+require("dotenv").config();
+
+function validateKWebhook(req, res, next) {
+  const validateHeader = req.headers["webhook-validator"];
+
+  if (process.env.WEBHOOK_VALIDATOR !== validateHeader) {
```

Comment:
What's saved in WEBHOOK_VALIDATOR? The URL of the FF manager?

Reply:
It's the new environment variable.